appinfo/app.php

```diff
@@ -20,26 +20,26 @@
 
 const APP_NAME = 'user_shibboleth';
 
-require_once OC_App::getAppPath(APP_NAME) . '/appinfo/bootstrap.php';
+require_once __DIR__.'/bootstrap.php';
```

Comment:
Is this state safe, i.e. "functional"? Or will it give execution/inclusion path.